.stickler.yml

```diff
@@ -10,3 +10,5 @@ files:
     ignore:
         - 'src/generated/*'
         - 'webroot/bower_components/*'
+        - 'dist/*'
+        - 'src/*'
```

Comment:
- [ ] src/ directory should not be ignored by the sticker. You can ignore the test file or move it to another directory. There are some linter errors in your index file that need to be fixed.